### PR TITLE
Add testing helper to create labeled namespaces

### PR DESCRIPTION
**Pull request by manno**
Also adds teardown all helper

[#165900520](https://www.pivotaltracker.com/story/show/165900520)

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -41,22 +41,3 @@ func NewDefaultConfig(fs afero.Fs) *Config {
 		Fs:                   fs,
 	}
 }
-
-// NewConfig returns a new Config for a manager of controllers
```
Comment (mudler) [inline pkg/config/config.go:45]: :thinking: I guess that will break tests now as https://github.com/cloudfoundry-incubator/quarks-utils/pull/42 added a couple to cover it.. sorry :smiley:

File: testing/machine/machine.go [hunk 2]
```diff
@@ -51,10 +51,16 @@ func NewMachine() Machine {
 
 // CreateNamespace creates a namespace, it doesn't return an error if the namespace exists
 func (m *Machine) CreateNamespace(namespace string) (TearDownFunc, error) {
+	return m.CreateLabeledNamespace(namespace, map[string]string{})
```
Comment (rohitsakala) [inline testing/machine/machine.go:54]: I think labels will always be empty here.

File: pkg/config/config.go (hunk 1 above)
Comment (manno) [reply pkg/config/config.go:45]: I saw that, however this function appears to be unused ...

Comment (mudler) [reply pkg/config/config.go:45]: Looks good indeed! I've checked that too when adding the test.. but then wasn't sure if was being consumed elsewhere

**Pull request merged**